File: DSA/NiharikaChand_590017697/Day1Question3.java
import java.util.*;
class Day1Question3
{
    public static int calSum(int arr[], int n)
    {
        int i, sum=0;
        for(i=0; i<n; i++)
        {
            sum = sum+arr[i];
        }
        return sum;
    }
    public static void main(String args[])
    {
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter size of array: ");
        int n = sc.nextInt();
        int arr[] = new int [n];
        int i, sum=0;
        for(i=0; i<n; i++)
        {
            System.out.println("Enter an element: ");
            arr[i] = sc.nextInt();
        }

        sum = calSum(arr, n);
        System.out.println("Sum of all elements: " + sum);

    }
}

File: DSA/Stuti_Jain_590012149/Day_1_Question_3.c
#include<stdio.h>
int sum, i, n;
int main() {
    printf("Enter the size of the array: \n");
    scanf("%d", &n);

    int arr[n];

    printf("Enter the elements of the array: \n");
    for(i=0; i<n; i++){
        scanf("%d", &arr[i]);
    }

    for(i=0; i<n; i++){
        sum+=arr[i];
    }

    printf("The sum of the elements of the array is: %d", sum);
    
    return 0;
}
